spec/api/v1/questions_spec.rb

```diff
@@ -89,45 +77,15 @@
           expect(response.body).to be_json_eql(question.send(attr.to_sym).to_json).at_path("question/#{attr}")
         end
       end
+    end
 
-      context 'attachments' do
-        it 'included in question object' do
-          expect(response.body).to have_json_size(1).at_path("question/attachments")
-        end
-
-        it 'contains attachment url' do
-          expect(response.body).to be_json_eql(attachment.file.url.to_json).at_path("question/attachments/0/url")
-        end
-      end
-
-      context 'comments' do
-        it 'included in question object' do
-          expect(response.body).to have_json_size(1).at_path("question/comments")
-        end
-
-        %w(id body commentable_type commentable_id user_id created_at updated_at).each do |attr|
-          it "contains #{attr}" do
-            expect(response.body).to be_json_eql(comment.send(attr.to_sym).to_json).at_path("question/comments/0/#{attr}")
-          end
-        end
-      end
+    def do_request(options = {})
+      get "/api/v1/questions/#{question.id}", params: { format: :json }.merge(options)
     end
   end
 
   describe 'POST /create' do
-    context 'unauthorized' do
-      it 'returns 401 status if there is no access token' do
-        post "/api/v1/questions", params: { format: :json, question: attributes_for(:question) }
-        expect(response.status).to eq 401
-      end
-
-      it 'returns 401 status if access token is invalid' do
-        post "/api/v1/questions", params: { 
-          format: :json, access_token: '1234', question: attributes_for(:question)
-        }
-        expect(response.status).to eq 401
-      end
-    end
+    it_behaves_like 'API Authenticable'
 
     context 'authorized' do
       let(:access_token) { create(:access_token) }
```

Comment:
[`it 'returns 201 status'`](https://github.com/RomanSerikov/qna/pull/14/files#diff-67708221346ca8ba120a716a03eace42L135).